packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -156,24 +156,22 @@ async function _getDashboardStats(app) {
   const dashboard = {}
 
   let hvSupportedVersions
+  let nHostsEol
   try {
     hvSupportedVersions = await app.apiMethods['xoa.getHVSupportedVersions']()
-  } catch (e) {
-    hvSupportedVersions = {
-      'Citrix Hypervisor': '>=8.2.0',
-      'XCP-ng': '>=8.2.0',
-    }
+    nHostsEol = 0
+  } catch (error) {
+    console.error(error)
```

Comment:
To avoid flooding the logs, you should handle `xoa.getHVSupportedVersions` not being defined (i.e. when XO is used from the sources).